src/model_bias_analysis.py

```diff
@@ -70,7 +71,7 @@ def add_subgroup_columns_from_text(df, text_column, subgroups):
     New column contains True if the text contains that subgroup term.
     """
     for term in subgroups:
-        df[term] = df[text_column].apply(lambda x: term in x)
+        df[term] = df[text_column].apply(lambda x: bool(re.search(r'\b{}\b'.format(term), x)))
```

Comment:
I believe this is still case-sensitive. I think you just need to add `flags` to the `re.search` call:

`bool(re.search(r'\b{}\b'.format(term), x, flags=re.IGNORECASE))`

Reply:
Thanks, fixed